python/master.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul 25 11:28:01 2019
+
+@author: marcdimeo
+"""
+
+import mygene
+import pandas as pd
+import numpy as np
+from sklearn import preprocessing
+import scipy.io as spio
+import os
+from scipy.stats.stats import pearsonr
+import matplotlib.pyplot as plt
+import seaborn as sns
+import plotly.io as pio
+import plotly.figure_factory as ff
+from scipy.spatial.distance import pdist, squareform
+#import plotly.graph_objects as go
+
+"""
+FUNCTIONS
+"""
+
+def read_csv_df(file):
```

Comment:
Example of docstring:

"""
read_csv_df takes in the path to a file with whitespace delimiters and replaces zero values with NaN.

INPUT:
  * file: path to the file

OUTPUT:
  * data: data with empty values as NaN.
"""